src/AppInstallerCLICore/Workflows/SettingsFlow.cpp

```diff
@@ -12,6 +12,33 @@ namespace AppInstaller::CLI::Workflow
     using namespace AppInstaller::Settings;
     using namespace AppInstaller::Utility;
 
+    namespace
+    {
+        struct AdminSettings
+        {
+            AdminSettings()
+            {
+                root["adminSettings"] = Json::ValueType::objectValue;

```

Comment:
> root["adminSettings"] = Json::ValueType::objectValue; [](http://example.com/codeflow?start=15&length=54)

Is the exported settings to be imported in Powershell DSC? Do we need a schema version or it's unlikely to change in the future?

Reply:
For DSC... not entirely. The admin settings resource will take a hashtable with something really similar to value of the adminSettings. We just needed a way to expose it without reading registry keys.

Also, because reasons (mostly testing and not having to special case powershell) I'm adding the path of the local user settings file to `winget settings export` . Also `winget --info` for completeness. 

But yes, I'm going to add a new schema.